events.html

```diff
@@ -43,8 +43,42 @@
                 <div class="col-8">
                     <div class="card mb-2">
                         <div class="card-body">                
-                            June 2021
-                            <h3>TBD</h3>
+                            Th, Jun 24 : 6:00 - 9:00, some location
+                            <h3>How technically-savvy VCs think about startups</h3>
+                            <hr/>
+                            <p>
+                                Our panelists are all venture capitalists who invest in technical startups. We will
+                                discuss the industry as they see it, the technologies they're excited about, advice
+                                for foundners and prospective founders in technical industries, and field your questions.
+                            </p>
+                            <p>
+                                Panelists:
+                            </p>
+                            <ul>
+                                <li>Sarah Catanzaro, Amplify Partners</li>
+                                <li>Naomi Ionita, Menlo Ventures</li>
+                                <li>Randall Lloyd, Grix</li>
+                                <li>Bryan Offut, Index Ventures</li>
+                                <li>Lee Edwards, Root Ventures (Moderator)</li>
+                            </ul>
+                            <p>
+                                Food will be served
+                            </p>
+                            <p>
+                                Venue is outdoors. Masks and distancing will follow state guidelines.<br/>
+                                <ul>
+                                    <li><strong>6:00</strong> Doors open, socializing, mingling, and snacking</li>
+                                    <li><strong>7:00</strong> Panel: How technically-savvy VCs think about startups</li>
+                                    <li><strong>7:30</strong> Panel Q&A</li>
+                                    <li><strong>7:50</strong> Mingle, socialize</li>
+                                    <li><strong>9:00</strong> Doors close</li>
+                                </ul>
+                            </p>
+                            <p>
+                                Admission is one PR accepted into <a href="wisdom.html">/wisdom</a>.<br/>
```

Comment:
```suggestion
```

On reflection, I don't think we want to gatekeep that strongly. We can encourage contributions at the meetup itself.